### ISSUE TITLE
Version tag is missing 

### ISSUE THREAD

**Issue by lennet** [hartbit/Yaap#1]
The installation guideline is currently not working because of the missing version tag. 

I get the following error: 

```
error: dependency graph is unresolvable; found these conflicting requirements:

Dependencies: 
    https://github.com/hartbit/Yaap.git @ 0.0.0..<1.0.0
```

**Comment by alexito4** [comment]
@lennet FYI you can point directly to master. 

```
.package(url: "https://github.com/hartbit/Yaap.git", .branch("master"))
```

**Comment by hartbit** [comment]
My bad, I made a typo in the documentation. The first released version is `1.0.0`, not `0.1.0`. It's now fixed. Thanks for the reporting the issue!
